frontend/src/icons/BranchIcon.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC, SVGAttributes } from 'react';
+export type Icon = SVGAttributes<HTMLOrSVGElement>;
+
+const BranchIcon: FC<Icon> = () => {
```

Comment:
If you don't collect any props and don't add them to attributes, the props will not work. For example, if we pass props to control stroke color, but don't collect and add them to the attributes, it will not work.

Reply:
@tusheer when developer will need to pass props to re-use the svg icon component he will pass props accordingly his need. I have just made a prototype of these html svg code to icon component.

Reply:
@tusheer should I merge now?

Reply:
@harunurkst  Yes, You can merge now.

Reply:
thanks